packages/alpha/src/types.ts

```diff
@@ -17,47 +17,64 @@ import {
 
 export * from '@cognite/sdk';
 
-export interface MonitoringTaskParametersCreate {
-  timeseriesExternalId: CogniteExternalId;
-  granularity?: string;
-  threshold: number;
+enum ModelExternalId {
+  Threshold = 'threshold',
+}
```

Comment:
please do not use enums they're quite painful for the consumer.

`paramName: "string_val1" | "string_val1"` is usually fine